content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md

```diff
@@ -1,10 +1,220 @@
 ---
-title: Fault tolerance
-description: TODO
+title: Fault Tolerance and Recovery
+description: Learn how TiKV recovers from failures
 menu:
     "5.1":
         parent: Features
         weight: 2
 ---
 
-TODO
+This page walks you through a simple demonstration of how TiKV recovers from failures and continues to provide services during failures.
+
+1. Starting a 6-node local cluster.
```

Comment:
```suggestion
1. Start a cluster with 6 TiKV nodes.
```

Reply:
The same mean to me.

Reply:
Referring https://www.cockroachlabs.com/docs/v21.1/demo-fault-tolerance-and-recovery.html#step-1-start-a-6-node-cluster

Reply:
Updated suggestion